src/runtime/object.cc

```diff
@@ -103,6 +103,8 @@ class TypeContext {
 
     if (static_tindex != TypeIndex::kDynamic) {
       // statically assigned type
+      VLOG(3) << "TypeIndex[" << static_tindex << "]: static: " << skey << ", parent "
+              << type_table_[parent_tindex].name;
```

Comment:
Perhaps DLOG is better? not so sure about the level 3 here

Reply:
why not level 3?

Reply:
DLOG is something that safely be removed under no debug mode, and may help certain cases to reduce binary size

Reply:
if we had `DLOG(3)` i'd agree with you. what do you think about adding that?

Reply:
DLOG was not enabled by default and do not come with the level, how if you think adding level is helpful, i have no objections :)

Reply:
actually on second thought--VLOG is debugging info, why is it not removed when `TVM_LOG_DEBUG` is unset? i don't see an INFO/WARNING setting that makes sense for this type of debug info.

Reply:
please double check, if that is the case we could be fine

Reply:
@mbs-octoml said that while VLOG isn't explicitly compiled out, it's done in such a way that the compiler determines it's dead-code. i think this is better in case a variable is only used in VLOG.

Reply:
The mechanism is identical to DLOG.

Reply:
OK, thanks for clarifying. no more comments on my end